docs/report.md

```diff
@@ -26,13 +26,51 @@ Our domain model consists of two data entities, which depict the attributes of a
+The use of our application is defined by a range of tabs, a **cheep box** and two different kinds of users - **authenticated** users and **unauthenticated** users. That is, users that are logged in and users that are not. The two kinds of users differ in the elements that they have access to on the site. 
+#### Unauthenticated
+![Chirp public timeline unauthenticated](https://github.com/ITU-BDSA23-GROUP12/Chirp/blob/363-design-and-architecture-user-activities/docs/images/unregistered%20chirp.png)
+
+The **unauthenticated** users have access to the **Public Timeline** tab, the **Register** tab and the **Login** tab.
+##### Public Timeline
+The **Public Timeline** contains a list of each (non-deleted) cheep ever sent, and is available to both authenticated and unauthenticated users. The cheeps are displayed on pages of 32 cheeps, and the pages can browsed between using the **next** and **previous** buttons on the button of the page.
+##### Register
+If the user has not registered before, clicking the **Register** tab redirects the user to a prompt for signing in to GitHub to continue to Chirp. After signing in with GitHub, the user is registered and logged in to Chirp, and is now an authenticated user. If the user have registered before, this information might still be stored in cookies managed by the browser, and will then not be asked to sign in with GitHub - they will simply be logged in right away and immediately turn into an authenticated user. Once authenticated, the user's GitHub username is also used as the user's Chirp username.
+Only unauthenticated users have access to the register tab.
+##### Login
+The **Login** tab does the exact same thing as the register tab - their behaviour is the same in every way.
+#### Authenticated
+![Chirp public timeline\label{Authenticated Public Timeline}](https://raw.githubusercontent.com/ITU-BDSA23-GROUP12/Chirp/main/docs/images/public%20timeline.png)
+The **authenticated** user has access to the **Public Timeline**, **My Timeline**, **Discover**, **Logout**, and **About Me** tabs.
+##### The follow mechanic
+An authenticated user has, attached to each cheep on the public timeline, a **follow** button. Uppon clicking this, the follow button is replaced by an **unfollow** button, and the user will be regarded as following the user that sent the particular cheep.
+##### My Timeline
+The **my timeline** tab is similar to the public timeline, but here only the user's own cheeps and cheeps written by users that the user is following are displayed. This tab is only accessible for authenticated users, and is only privately accessible.
+##### Discover
+Let the user of the application be A.
+The **discover** tab contains the latest cheep of each user B that is deemed interesting for A. B is deemed interesting if at least two users followed by A are following B. Here, A can browse through users that might be more relevant to A. The discover tab is only accessible to authenticated users.
```

Comment:
Maybe mention the ordering of cheeps. It is not cronological, but how relevant i might be. If User b is followed by 4 user followed by A, and user C is followed by 3 users followed by user A, the cheep from user B will be at the top
Also only people that A dont already follow be shown in discover

Reply:
Nicely explained, im thinking maybe a small diagram could be nice to understand text better